changelogs/fragments/6773-proxmox_kvm-restarted-state-bug-fix.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - proxmox_kvm - ``restarted`` state did not actually restart a vm in some vm configurations. The state now uses the Proxmox reboot endpoint instead of calling the ``stop_vm`` and ``start_vm`` functions. (https://github.com/ansible-collections/community.general/pull/6773)
```

Comment:
```suggestion
  - proxmox_kvm - ``restarted`` state did not actually restart a VM in some VM configurations. The state now uses the Proxmox reboot endpoint instead of calling the ``stop_vm`` and ``start_vm`` functions (https://github.com/ansible-collections/community.general/pull/6773).
```